### DISCUSSION TITLE
HEIC thumbnails not showing when browser requests 250x250px in "Files"

### DISCUSSION THREAD

**Discussion by fumped** [nextcloud/all-in-one#3325] [Questions] [answered]
I created a thread on the support forums but I believe this might be the correct place to ask since it seems like a bug and not a configuration error by me, please correct me if I'm wrong :) 

Please read:
https://help.nextcloud.com/t/all-in-one-error-400-imaginary-unable-to-serve-250x250-thumbnails/169693

In summary: I run identical OS and browser on my laptop and desktop. The desktop has a 3440x1440 display and when using 100% scaling the browser requests 250x250px thumbnails from the server which causes tons of "Error 400 bad requests.
Messing around with the thumb requests using the inspector it seems that the smallest thumb size the browser can successfully request is 257x257px.
Workaround for now is to run my browser in 110% scaling and the thumbs load successfully.

**Comment by szaimen** [comment, marked as answer]
Hi, this needs to be fixed in nextcloud/server.

Please open a new bug report here if there shouldnt be one already: https://github.com/nextcloud/server/issues
